torchensemble/utils/operator.py

```diff
@@ -51,3 +53,21 @@ def pseudo_residual_regression(target, output):
         raise ValueError(msg.format(target.size(), output.size()))
 
     return target - output
+
+
+def majority_vote(outputs: List[torch.Tensor]) -> torch.Tensor:
+    """Compute the majority vote for a list of model outputs.
+    outputs: list of length (n_models)
+    containing tensors with shape (n_samples, n_classes)
+    majority_one_hots: (n_samples, n_classes)
+    """
+
+    assert (
```

Comment:
The `assert` clause is not recommended in the code for production, therefore, it would be better to use `raise ValueError` here ;-)